app/models/review.rb

```diff
@@ -12,6 +12,15 @@ class Review < ApplicationRecord
     rejected: 2
   }, _prefix: true
 
+  def booking
+    request&.booking
+  end
+
+  def room_type
+    request&.room_availability_requests&.first
+          &.room_availability&.room&.room_type
```

Comment:
```suggestion
    request&.room&.room_type
```
Chỗ này e trỏ đến room luôn cho gọn nha
Tương tự cho những chỗ khác